yarpl/include/yarpl/flowable/FlowableOperator.h

```diff
@@ -346,7 +346,13 @@ class SkipOperator : public FlowableOperator<T, T> {
      void request(int64_t delta) override {
        if (firstRequest_) {
          firstRequest_ = false;
-         delta = delta + offset_;
+
+         // TODO: This should be helper like `clampAdd` or something.
+         if (delta <= std::numeric_limits<int64_t>::max() - offset_) {
```

Comment:
Hmm it does, but only for `std::atomic<int64_t>`, and I have an `int64_t` here.  I could add another overload however.